.gitignore

```diff
@@ -7,4 +7,5 @@ venv/
 **/.pytest_cache/
 __pycache__/
 *.db3-*
-
+log/
+install/
```

Comment:
I would highly recommend to avoid unnecessary or not related changes to solve the issue. besides, why do we have these directories in `.gitignore` in this repository?

Reply:
I am working with vscode, and if i do not add it inside it it keep proposing to commit it. But i agree it is not part of the patch

Reply:
in that case, your global `.gitignore` should be set with this? and it works better for you. besides, not everyone is using vscode, i think that is user dependent setting, not repository global setting.